python/railjson_generator/railjson_generator/schema/infra/infra.py

```diff
@@ -87,7 +87,7 @@ def make_rjs_operational_points(self):
                 parts=parts_per_op[op.label],
                 extensions={  # pyright: ignore[reportCallIssue] - 'extensions' exists but is registered through 'register_extension'
                     "sncf": infra.OperationalPointSncfExtension(
-                        ci=0,
+                        ci=op.ci,
```

Comment:
We might not need to provide `ci` explicitly. https://fr.wikipedia.org/wiki/Code_r%C3%A9seau_UIC


```suggestion
                        ci=int(str(op.ci)[2:]), # remove two first digits of UIC code
```

If we apply this change, we also need to adapt the provided UIC codes.

Reply:
Done.